release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -951,6 +951,14 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
 
 * Previously, the BMC driver IPMI was not supported for a secure UEFI boot. This resulted in an unsuccessful boot. This fix adds a validation check to ensure that `UEFISecureBoot` mode is not used with bare-metal drivers. As a result, a secure UEFI boot is successful. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011893[*BZ#2011893*])
 
+* With this update, the 4.8 UPI template is updated from version 3.1.0 to 3.2.0 in order to match the Ignition version. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1949672[*BZ#1949672*])
+
+* Previously, when asked to mirror the contents of the base registry, the {product-title} installer would exit with a validation error, citing incorrect `install-config` file values for `imageContentSources`. With this update, the installer now allows `imageContentSources` values to specify base registry names and the installer no longer exits when specifying a base registry name. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1960378[*BZ#1960378*])
+
+* Previously, the UPI ARM templates are attaching an SSH key to the virtual machine (VM) instances created. As a result, the creation of the VMs fails when the SSH key provided by the user is the `ed25519` type. With this update, the creation of the VMs succeeds regardless of the type of the SSH key provided by the user. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1968364[*BZ#1968364*])
+
+* Previously, after successfully creating a `aws_vpc_dhcp_options_association` resource, AWS might still report that the resource does not exist. Consequently, the AWS Terraform provider fails the installation. With this update, you can retry the query of the `aws_vpc_dhcp_options_association` resource for a period of time after creation until AWS reports that the resource exists. As a result, installations succeed despite AWS reporting that the `aws_vpc_dhcp_options_association` resource does not exist. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032521[*BZ#2032521*])
```

Comment:
I would remove Previously, here. It throws off the tense of rest of the sentence.